Dockerfile

```diff
@@ -23,9 +23,10 @@ COPY internal/controller/ internal/controller/
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
 RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# Use alpine to have the smallest image possible while being able to use a required package for the code 
```

Comment:
```suggestion
# Use alpine instead of distroless to have the smallest image possible while being able to use a required package for the code (htpasswd)
# TODO(atlas): Move back to gcr.io/distroless/static:nonroot once we do not need htpasswd
```